app/angular/src/client/preview/angular-beta/RenderNgAppService.ts

```diff
@@ -19,14 +21,75 @@ import { isComponentAlreadyDeclaredInModules } from './NgModulesAnalyzer';
 export class RenderNgAppService {
   private static instance: RenderNgAppService;
 
+  public static SELECTOR_STORYBOOK_WRAPPER = 'storybook-wrapper';
+
   public static getInstance() {
     if (!RenderNgAppService.instance) {
       RenderNgAppService.instance = new RenderNgAppService();
     }
     return RenderNgAppService.instance;
   }
 
-  public static SELECTOR_STORYBOOK_WRAPPER = 'storybook-wrapper';
+  public static getNgModuleMetadata = (
+    {
+      storyFnAngular,
+      parameters,
+    }: {
+      storyFnAngular: StoryFnAngularReturnType;
+      parameters: Parameters;
+    },
+    storyProps$: Subject<ICollection>
+  ): NgModule => {
+    const {
+      component: storyComponent,
+      props,
+      styles,
+      template,
+      moduleMetadata = {},
+    } = storyFnAngular;
+
+    if (storyComponent) {
+      deprecate(
```

Comment:
You need to declare a warning function at the top of the file like in the order code examples. Otherwise the deprecation warning prints every time you call it, rather than just on the first call